androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -48,6 +49,11 @@ public class TwaLauncher {
 
     private static final int DEFAULT_SESSION_ID = 96375;
 
+    private static final String EXTRA_STARTUP_UPTIME_MILLIS =
+            "org.chromium.chrome.browser.customtabs.trusted.STARTUP_UPTIME_MILLIS";
+
+    private static final String EXTRA_ABH_VERSION = "org.chromium.chrome.browser.ABH_VERSION";
+
```

Comment:
As in https://chromium-review.googlesource.com/c/chromium/src/+/6520581/comment/5794ab53_0454b740/, it might be better to use lower case naming conventions for the Intent strings.

Reply:
As the corresponding variable in the CL moved to a different file, decided to leave the letters capitalized following its convention.